docs/user-guide/src/bmo/provisioning.md

```diff
@@ -108,12 +108,7 @@ provisioning](./live-iso.md) that works differently from a normal one.
 
 ### Notes on checksums
 
-Unlike Ironic itself, Metal3 currently assumes the checksum algorithm to be MD5
-when no `checksumType` value is provided. Since more secure algorithms, such as
-SHA256 or SHA512, are popular nowadays, care must be taken to provide the
-correct `checksumType`. The value of `auto` will make Ironic detect the
-checksum type from its length and will become the default in the next version
-of the BareMetalHost API.
+Setting `checksumType` to `auto` or leaving it empty prompts Ironic to automatically detect the checksum type based on its length.
```

Comment:
Please limit line length to 80.

I would also not delete the note on the baremetalhost API change. API change != this change.